src/python/pants/backend/go/util_rules/cgo.py

```diff
@@ -306,8 +299,11 @@ async def make_cgo_compile_wrapper_script(
                     path="wrapper",
                     content=textwrap.dedent(
                         """\
-                sandbox_root="$(/bin/pwd)"
-                args=("${@//__PANTS_SANDBOX_ROOT__/$sandbox_root}")
+                export sandbox_root="$(/bin/pwd)"
+                ln -s "${sandbox_root}" __pants_sandbox_root__
```

Comment:
What is this symlink for? I don't see where it's used?

Reply:
Ugh leftover from the original PR that I was cleaning up. Good spotting it!

Reply:
Context: I had originally tried to use the symlink to solve the problem solved by the next PR (not this one). Missed removing this for this PR. Grr..